services/gafaelfawr/values-usdfdev.yaml

```diff
@@ -0,0 +1,232 @@
+replicaCount: 2
+
+# Use the CSI storage class so that we can use snapshots.
+redis:
+  persistence:
+    storageClass: "wekafs--sdf-k8s01"
+
+config:
+  databaseUrl: "postgresql://gafaelfawr@postgres.postgres/gafaelfawr"
```

Comment:
Advance warning that we're going to drop support for the internal postgres chart in a later version of Phalanx, and this should really use infrastructure-provided PostgreSQL.